ludwig/data/dataframe/base.py

```diff
@@ -46,6 +46,10 @@ def map_objects(self, series, map_fn, meta=None):
     def map_partitions(self, series, map_fn, meta=None):
         raise NotImplementedError()
 
+    @abstractmethod
+    def try_map_batches(self, series, map_fn, batch_format="pandas", meta=None):
```

Comment:
I would rename to just `map_batches`, and maybe drop the `batch_format` param, since it would be difficult to make it work with other backends.